Homework/week-2/program.js

```diff
@@ -0,0 +1,9 @@
+var arguments = process.argv;
+
+var sum = 0;
+for(var i = 0; i < arguments.length; i++){
```

Comment:
Here you could say for (var i = 2; i < arguments.length; i++) instead of having to do the check below.